modules/wr-mil/x86/wrmil-int-f50sync-br3_start.sh

```
@@ -40,7 +40,7 @@ saft-wbm-ctl $SDGW -x
 echo -e WRF50: load firmware 
 eb-fwload $TRGW u 0x0 wrf50.bin
 
-echo -e WRF50: configure firmware for gateway $NGW
+echo -e WRF50: configure firmware for gateway
```

Shellcheck (0.11.0)

[warning] In POSIX sh, echo flags are undefined.

(SC3037)